packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_p2wpkh.test.ts

```diff
@@ -79,17 +73,54 @@ describe('createDeFiTx()', () => {
     expect(unmapped.hex).toStrictEqual('001234')
   })
 
+  it('balance should be deducted accordingly based on spent on DfTx - higher val', async () => {
```

Comment:
This test was failing in my local, unable to find the cause. Could use extra pair of 👀